src/user-metadata/user-metadata.service.ts

```diff
@@ -154,7 +154,7 @@ export class UserMetadataService {
                 Assessor: {
                     select:{
                         userId: true,
-                        userName: true,
+                        // userName: true,
```

Comment:
Why is this commented out?

Reply:
Removed previously my server was not running